src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -182,10 +206,17 @@ export class QuickBar extends LitElement {
         .twoline=${Boolean(item.altText)}
         .item=${item}
         index=${ifDefined(index)}
-        graphic="avatar"
+        graphic="icon"
```

Comment:
Using "avatar" created wayyy oversized icons for the navigation command items that use svg iconPath.